app/Http/Controllers/SearchController.php

```diff
@@ -85,7 +86,13 @@ public function search(Request $request)
 		 * Photos
 		 */
 		// for now we only look in OUR pictures
-		$query = Photo::where('owner_id', '=', $id);
+		$query = Photo::where('id', '>', 0);
```

Comment:
What is the purpose of this check? Aren't ids always greater than 0 ?

Reply:
As I said:

> PS: I do not like query builders too much ;-)

I added the first dummy condition to get the brackets around the first non-dummy WHERE-condition with the OR-components, ie. getting from "here" ...

`select * from `photos` where `id` > ? and `owner_id` is null or `public` >= ? and (`title` like ? or `description` like ? or `tags` like ?) [9,0,"%privat%","%privat%","%privat%"] `

... to "there":

`select * from `photos` where `id` > ? and (`owner_id` is null or `public` >= ?) and (`title` like ? or `description` like ? or `tags` like ?) [0,0,"%privat%","%privat%","p%rivat%"] `